src/lib/components/loading/loading.js

```diff
@@ -21,28 +24,68 @@ export default class Loading extends PureComponent {
       return (
         <div className={styles[`theme-${theme}`]}>
           <div className={styles.loading}>
-            <svg className={styles.loadingSvg} xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1000 1000">
+            <svg
+              className={styles.loadingSvg}
+              xmlns="http://www.w3.org/2000/svg"
+              viewBox="0 0 1000 1000"
+            >
               <defs>
                 <linearGradient id="gradient" x1="0%" y1="0%" x2="0%" y2="100%">
-                  <stop offset="0%" stop-color="#9135F0" />
-                  <stop offset="100%" stop-color="#5eaefd" />
+                  <stop offset="0%" stopColor="#9135F0" />
+                  <stop offset="100%" stopColor="#5eaefd" />
                 </linearGradient>
               </defs>
-              <title>Loader</title>
+              <title>… Loading …</title>
```

Comment:
I just changed the title and the `stopColor` – the rest are formatting changes, sorry